project/Settings.scala

```diff
@@ -577,14 +586,6 @@ object Settings {
     // We put our classes on scalac's `javabootclasspath` so that it uses them
     // when compiling rather than the definitions from the JDK.
     recompileAllOrNothingSettings,
-    Compile / scalacOptions := {
```

Comment:
Just out of curiosity, why this was needed, and now we can remove it?

Reply:
The history tracks back to #605, however I'm not sure why it's no longer an issue. Maybe it was fixed in Scala 2.12